torch_xla/csrc/runtime/computation_client.h

```diff
@@ -447,6 +447,9 @@ class ComputationClient {
   // after the last ':' character of the device string.
   static int64_t GetDeviceOrdinal(const std::string& device);
 
+  virtual void SetCustomCompileOptions(
```

Comment:
Document what happens if this is called multiple times? (Are the options overwriting existing options or appending to them? If the latter, the function should be named Add*)